openprescribing/frontend/management/commands/import_ppu_savings.py

```diff
@@ -64,7 +68,8 @@ def make_merged_table_for_month(
     """
     cases = []
     seen = Set()
-    df = pd.read_csv(substitutions_csv)
+    r = requests.get(SUBSTITUTIONS_SPREADSHEET)
+    df = pd.read_csv(StringIO(r.content))
```

Comment:
It's just come to me that `pandas.read_csv` can take a URL, saving you a couple of lines of code!!!!!!